src/logic.js

```diff
@@ -83,8 +85,10 @@ define([
                         }
                     } else if (node instanceof xpath.models.XPathFuncExpr) {
                         this._addInstanceRef(node);
+                    } else if (node instanceof xpath.models.HashtagExpr) {
+                        hashtags.push(node);
                     }
-                    children = node.getChildren();
+                    children = node.getChildren ? node.getChildren() : [];
```

Comment:
What kind of node does not have a `getChildren` method? I assume from this context it's a `HashtagExpr`? If that's the case, it would be good to implement `HashtagExpr.getChildren()` to return `[]`.